image/Dockerfile-stack

```diff
@@ -1,35 +1,34 @@
 FROM node:12
 
 ENV APPSODY_PROJECT_DIR=/project
-ENV APPSODY_MOUNTS=/:/project/user-app
-ENV APPSODY_DEPS=/project/user-app/node_modules
+ENV APPSODY_MOUNTS=.:/project/function
+ENV APPSODY_DEPS=/project/function/node_modules
 
-ENV APPSODY_WATCH_DIR=/project/user-app
-ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/node_modules
+ENV APPSODY_WATCH_DIR=/project/function
+ENV APPSODY_WATCH_IGNORE_DIR=/project/function/node_modules
 ENV APPSODY_WATCH_REGEX="^.*.js$"
 
+ENV APPSODY_TEST="npm test && npm test --prefix function"
+ENV APPSODY_TEST_ON_CHANGE=""
+ENV APPSODY_TEST_KILL=false
+
 ENV APPSODY_RUN="npm start"
-ENV APPSODY_RUN_ON_CHANGE="npm start"
-ENV APPSODY_RUN_KILL=true
+ENV APPSODY_RUN_ON_CHANGE="$APPSODY_RUN"
 
 ENV APPSODY_DEBUG="npm start --node-options --inspect=0.0.0.0"
-ENV APPSODY_DEBUG_ON_CHANGE="npm start --node-options --inspect=0.0.0.0"
-ENV APPSODY_DEBUG_KILL=true
```

Comment:
Same question as above. It's not clear that removing these results in the same behavior.

Reply:
You're very right that removing the first command (the on_change) would result in different behavior, but it is still there, just parameterized to make it explicit that it's an alias of the debug command.  The second line, kill, is indeed the default as stated by the docs, but I added it back anyway as the new line 21.